src/rHelpProvider.ts

```diff
@@ -284,16 +291,26 @@ export class AliasProvider {
         this.allPackageAliases = null; 
 		const lim = '---vsc---'; // must match the lim used in R!
 		const re = new RegExp(`^.*?${lim}(.*)${lim}.*$`, 'ms');
-        const cmd = `${this.rPath} --silent --no-save --no-restore --slave -f "${this.rScriptFile}"`;
+        const tempDir = fs.mkdtempSync(path.join(os.tmpdir(), 'vscode-R-aliases-'));
```

Comment:
Is there a reason to create a temp dir rather than a simple temp file?

Reply:
Just the convenience of using `fs.mkdtempSync` to ensure we have a valid, unique path. Using a static filename might cause problems if multiple vscode instances are running simultaneously and generating a random filename (and checking its availability) would make things more complicated.

Reply:
Thanks for the clarification!